R/utilities-simulation-results.R

```diff
@@ -83,6 +83,75 @@ getOutputValues <- function(simulationResults, quantitiesOrPaths, individualIds
   return(output)
 }
 
+#' Export values that can be plugged directly into the TLF lib (TODO)
+#'
+#' @export
+getOutputValuesTLF <- function(simulationResults, population, quantitiesOrPaths = NULL, individualIds = NULL) {

```

Comment:
@PavelBal @pchelle @abdullahhamadeh 
This method works but is suboptimal
It takes about 15 sec to retrieve the value for 1000 individuals and one output! 
How can we profile this? Also are there any obvious place were we can optimize?